sgkit/stats/aggregation.py

```diff
@@ -167,6 +200,53 @@ def count_variant_alleles(ds: Dataset, merge: bool = True) -> Dataset:
     return conditional_merge_datasets(ds, new_ds, merge)
 
 
+def count_cohort_alleles(ds: Dataset, merge: bool = True) -> Dataset:
+    """Compute per cohort allele counts from genotype calls.
+
+    Parameters
+    ----------
+    ds
+        Genotype call dataset such as from
+        `sgkit.create_genotype_call_dataset`.
+    merge
+        (optional)
+        If True (the default), merge the input dataset and the computed
+        output variables into a single dataset, otherwise return only
+        the computed output variables.
+        See :ref:`dataset_merge` for more details.
+
+    Returns
+    -------
+    Dataset containing variable `call_allele_count` of allele counts with
+    shape (variants, cohorts, alleles) and values corresponding to
+    the number of non-missing occurrences of each allele.
```

Comment:
I'm wondering if this shouldn't also return a (variants, cohorts) array that contains the number of present samples used in each aggregation.  It doesn't do it now, but if `count_call_alleles` returned -1 for all alleles when any (or potentially all) input calls were missing then I can see a count of the samples with non-negative allele counts being a useful downstream denominator.

Is that not important with the stats functions in `popgen.py` now?  It may make sense to put a warning on those functions and log an issue related to supporting it.  It seems like only ignoring the missing values during the original allele count must introduce some kind of bias through all those aggregations.

Reply:
Good point. I haven't thought about missingness with respect to the meaning of the functions though. Opened #290